### PR TITLE
Updating for the 4.5.6 Release.

### PR DESCRIPTION
**Pull request by Chr1st0ph3rTurn3r**

### CODE REVIEW

File: docs/release_notes_128t_4.5.md [hunk 1]
```diff
@@ -3,6 +3,36 @@ title: 128T 4.5 Release Notes
...
+
+- **I95-37368 Allow Reset of Asset ID Field:** An asset-id can be reset from the Asset ID field in the GUI.
+------
+- **I95-38378 Salt-minion config broken after enabling asset connection resiliency:** The salt-minion configuration file now loads at the time of a migration operation, to ensure the latest version of the configuration is available. 
+------
+- **I95-38815 Set Adjacency Hostnames as DPU on Config** Dynamic peer hostnames shared via DPU are now properly handled internally and no longer produce DNS queries.
+------
+- **I95-38843 Highway crash shortly after upgrade to 4.5.4** Removed an unused component that was causing reverse key collisions for multi-hop scenarios. 
+------
+- **I95-39106 DHCP alarms are not cleared when config is changed to static:** We now handle the resulting empty string in alarm code after an interface is made static.
```
Comment (robert-patterson) [inline docs/release_notes_128t_4.5.md:18]: The empty string is an implementation detail.

**I95-39106 DHCP alarms are not cleared when config is changed to static:** Clear stale alarms from unresolved DHCP enabled interfaces when reconfigured to use static address.

File: docs/release_notes_128t_4.5.md [hunk 2]
```diff
@@ -3,6 +3,36 @@ title: 128T 4.5 Release Notes
...
 
+## Release 4.5.6
+
+### Resolved Issues
+
+- **I95-37368 Allow Reset of Asset ID Field:** An asset-id can be reset from the Asset ID field in the GUI.
+------
+- **I95-38378 Salt-minion config broken after enabling asset connection resiliency:** The salt-minion configuration file now loads at the time of a migration operation, to ensure the latest version of the configuration is available. 
+------
+- **I95-38815 Set Adjacency Hostnames as DPU on Config** Dynamic peer hostnames shared via DPU are now properly handled internally and no longer produce DNS queries.
```
Comment (tjbresnahan) [inline docs/release_notes_128t_4.5.md:14]: This is specifically for unresolved hostnames, which were not properly handled previously.

`Unresolved dynamic peer hostnames are now properly handled via DPU and no longer produce DNS queries.`

File: docs/release_notes_128t_4.5.md [hunk 3]
```diff
@@ -3,6 +3,36 @@ title: 128T 4.5 Release Notes
...
+------
+- **I95-39189 CVE Medium: CESA-2020:5566 openssl:** The openssl package has been updated to the latest version. 
+------
+- **I95-39269 Interface Stats on Router Page do not show why an interface is up/down:** The GUI now indicates whether an interface is Admin down or unplugged. 
+------
+- **I95-39319 Improve HMAC action performance via reusing context:** Changed lifetime of HMAC context to no longer be per packet, providing a much more efficient way of interacting with the OpenSSL HMAC library.
+------
+- **I95-39352 128T ISO boot logo has old 128T icon:** The splash screen has been updated to display the Juniper logo. 
+------
+- **I95-39378 5.1 Container Conductor failed resulting in an un-usable PCLI and missing system stats:** This issue has been resolved.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_4.5.md:34]: Rename title to "PCLI may return partial or empty results from some commands"

**Review by tjbresnahan — APPROVED** [review]

**Review by robert-patterson — DISMISSED** [review]

File: docs/release_notes_128t_4.5.md [hunk 4]
```diff
@@ -5,33 +5,104 @@ sidebar_label: 4.5
 
 ## Release 4.5.6
 
+### New Features and Improvements
+
+- **[I95-35051 Provide a way to generate & stage all auto-generated configuration](cli_reference.md#create-config-autogenerated):** The `create-config autogenerated` command has been added, which forces re-generation of all automatically generated configuration items, and stages the configuration changes into the current candidate configuration. This command serves only to aid in debugging, and allows you to validate, inspect, and make edits, without committing the changes.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_4.5.md:10]: Was the reference to `create-config-autogenerated` carried over from the 5.1 release notes? I only see the one file changed in this PR.
On another note, you will need to update the `about_releases.md` file to update the release date of 4.5.6

**Review by MichaelBaj — CHANGES_REQUESTED** [review]

File: docs/release_notes_128t_4.5.md (hunk 4 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_4.5.md:10]: Yes it was carried over from the 5.1 release notes.

**Review by MichaelBaj — DISMISSED** [review]

File: docs/release_notes_128t_4.5.md [hunk 5]
```diff
@@ -80,6 +82,8 @@ sidebar_label: 4.5
 ------
 - **I95-39036 dns-config=automatic should be allowed when using PPPoE:** Resolved an issue where management over pppoe interface does not allow dns-config mode to be configured.
 ------
+- **I95-39069 Highway crash in one thread when another thread is being modified:** This issue has been resolved.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_4.5.md:85]: Change statement to:
"**I95-39069 Highway crash when committing configuration:** Multi-threading race condition when processing multicast"

File: docs/release_notes_128t_4.5.md [hunk 6]
```diff
@@ -94,15 +98,25 @@ sidebar_label: 4.5
 ------
 - **I95-39190 CVE Medium: CESA-2021:0153 dnsmasq:** Upgraded dnsmasq to version 2.76-16.el7_9.1 as a remediation for CESA-2021:0153.
 ------
+- **I95-39192 BFD not able to recover after one-off source-nat changes on both directions:** BFD Session code has been modified to recover in this scenario. 
+------
 - **I95-39239 Event persistence gets stuck on a high number of events:** This issue has been resolved. 
 ------
+- **I95-39248 Session Migration failed due to Session Modification:** The update logic has been modified to handle a metadate update during the migration process.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_4.5.md:105]: typo. metadate should be metadata

**Review by MichaelBaj — CHANGES_REQUESTED** [review]

File: docs/release_notes_128t_4.5.md (hunk 4 above)
Comment (jeffmart) [reply docs/release_notes_128t_4.5.md:10]: I believe this is addressed so resolving conversation.

File: docs/release_notes_128t_4.5.md (hunk 1 above)
Comment (jeffmart) [reply docs/release_notes_128t_4.5.md]: This looks to be resolved.

File: docs/release_notes_128t_4.5.md (hunk 2 above)
Comment (jeffmart) [reply docs/release_notes_128t_4.5.md]: This looks to be resolved.

File: docs/release_notes_128t_4.5.md [hunk 7]
```diff
@@ -3,6 +3,36 @@ title: 128T 4.5 Release Notes
 sidebar_label: 4.5
 ---
 
+## Release 4.5.6
+
+### Resolved Issues
+
+- **I95-37368 Allow Reset of Asset ID Field:** An asset-id can be reset from the Asset ID field in the GUI.
+------
+- **I95-38378 Salt-minion config broken after enabling asset connection resiliency:** The salt-minion configuration file now loads at the time of a migration operation, to ensure the latest version of the configuration is available.
```
[the remark this answers is not in the document]
Comment (jeffmart) [reply docs/release_notes_128t_4.5.md]: This looks to be resolved.

File: docs/release_notes_128t_4.5.md [hunk 8]
```diff
@@ -3,6 +3,36 @@ title: 128T 4.5 Release Notes
 sidebar_label: 4.5
 ---
 
+## Release 4.5.6
+
+### Resolved Issues
+
+- **I95-37368 Allow Reset of Asset ID Field:** An asset-id can be reset from the Asset ID field in the GUI.
```
[the remark this answers is not in the document]
Comment (jeffmart) [reply docs/release_notes_128t_4.5.md]: This looks to be resolved.

File: docs/release_notes_128t_4.5.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (jeffmart) [reply docs/release_notes_128t_4.5.md]: This looks to be resolved.

File: docs/release_notes_128t_4.5.md (hunk 3 above)
Comment (jeffmart) [reply docs/release_notes_128t_4.5.md]: This looks resolved.

File: docs/release_notes_128t_4.5.md (hunk 6 above)
Comment (jeffmart) [reply docs/release_notes_128t_4.5.md]: Looks to be resolved with the latest changes

File: docs/release_notes_128t_4.5.md (hunk 5 above)
Comment (jeffmart) [reply docs/release_notes_128t_4.5.md]: This looks to be resolved with the latest update

**Review by jeffmart — APPROVED** [review]

**Review by MichaelBaj — APPROVED** [review]

**Pull request merged**